src/components/styles/books.css

```diff
@@ -0,0 +1,157 @@
+.books-container {
+  padding: 2.313rem 6.25rem;
+}
+
+.book-card {
+  height: 10.625rem;
+  margin: 1rem 0;
+  border-radius: 4px;
+  border: solid 1px var(--borderLines);
+  background-color: var(--white);
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  list-style: none;
+  padding: 1.688rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
- [ ] Please kindly make sure your design match exactly the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378), the colors, font,... 🙃 🙃

Your design 
<img width="853" alt="Screen Shot 2022-08-17 at 11 46 49" src="https://user-images.githubusercontent.com/74501523/185089740-c5428ea8-ce22-455a-89a0-8db316bad1a8.png">

Zeplin design
<img width="704" alt="Screen Shot 2022-08-17 at 11 46 13" src="https://user-images.githubusercontent.com/74501523/185089810-f7159adf-de13-450a-b8ae-376283b97b17.png">